test/probes/mongodb-core.test.js

```diff
@@ -129,6 +129,8 @@ function makeTests (db_host, host, isReplicaSet) {
       }
     })
 
+    ao.logLevel = 'error,warn,debug,patching'
```

Comment:
in other tests you save the previous loglevel and reset it later. Needed or not needed here?
why this selection of levels, when only 'debug' is used in this file? Is there logging happening somewhere else. If so, why is 'info' excluded?

Reply:
`error,warn` is the default and I want to keep them active. `patching` is used when a probe is loaded by the javascript `require` function (more precisely, our replacement of the native `require` function). `info` is not in the default set so it's not included.

logging overall needs to be scrubbed but i couldn't really have done it before i understood the use cases.